coops-ui/src/views/StandaloneDirectorsFiling.vue

```diff
@@ -844,4 +853,9 @@ h2 {
     margin-left: 0.5rem;
   }
 }
+
+.complianceDialogMsg {
+  font-size:16px;
```

Comment:
Please change size to "1rem" (which in our case is 16px).

For fonts especially, use rem units to allow the browser to resize elements consistently.